src/Catalyst.Cli.Tests/UnitTests/Commands/Response/GetDeltaResponseTests.cs

```diff
@@ -46,7 +46,7 @@ public void GetDeltaResponse_Can_Get_Output()
             //Arrange
             var deltaResponse = new GetDeltaResponse {Delta = new Delta()};
             var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
-            var getDeltaCommand = new GetDeltaCommand(commandContext, _logger);
+            new GetDeltaCommand(commandContext, _logger);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'GetDeltaCommand' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)